spec/support/mock_api.rb

```diff
@@ -1,56 +1,48 @@
 module AnsibleTowerClient
   class MockApi
     class Response
-      attr_reader :body
-      def initialize(body)
+      attr_reader :body, :url
+
+      def initialize(body, url = nil)
         @body = body
+        @url  = url
       end
     end
 
+    RESPONSES = {
+      'ad_hoc_commands'             => 'AdHocCommand',
```

Comment:
too bad (or good) that we are not in rails, otherwise, this hash can be replaced by `classify`

Reply:
considering that previously it was `case` chain instead of simple hash table / associative array, I believe it's a huge improvement anyway :)